services/web-server/src/graphql/WorkerManager.graphql

```diff
@@ -16,6 +16,25 @@ type WorkerManagerWorkerTypeSummary {
     provider: String!
 }
 
+type WorkerManagerWorker {
+    workerId: String!
+    workerGroup: String!
+    workerAge: DateTime
+
+    latestTaskRun: TaskRun
+
+    quarantineUntil: DateTime
+
+    recentErrors: Int
+
+    providerId: String!
+    workerType: String!
+}
```

Comment:
Do we know which properties will require an extra request and which won't? If `providerId` and `workerType` don't trigger an extra request, they could be brought up below `workerAge` without the newline.

Reply:
Not yet (at least, I'm not concentrating on that yet). I organized this with spaces to have the following structure:

`new fields from worker manager`
`data from queue`
`I'm not even sure about this one`
`two fields that "roll over" from the previous table`

It's more for my convenience than for anything. If you like, I can add this to the clean up ticket to remove all the white spaces later

Reply:
Adding it to the clean up ticket works for me.